src/config/routes.ts

```diff
@@ -5,6 +5,7 @@ export const AppRoutes = {
   load: '/load',
   index: '/',
   home: '/home',
+  demo: '/demo',
```

Comment:
I added this for testing purposes. I will remove it, as well as `demo.tsx`, before merging.